crosslink/cmd/root.go

```diff
@@ -31,7 +31,8 @@ type commandConfig struct {
 	pruneCommand cobra.Command
 }
 
-func (c *commandConfig) buildCommands() {
+func newCommandConfig() *commandConfig {
+	c := new(commandConfig)
```

Comment:
```suggestion
	c := &commandConfig{
		runConfig: cl.DefaultRunConfig(),
	}
```

This way there is no point at which this value is not usable.